### PR TITLE
Ports MacHac's rev flash rework from Yogstation

### PR DESCRIPTION
**Pull request by Super3222**
Includes some revisions.
- Rev flash will no longer make a "flash sound" when on CD
- Rev heads will be told that the flash is on CD.
- Non-rev heads using a flash on CD will be told there's a blinking light going on and off
- Rev heads will be informed how to use rev hijackers when the round starts.

### CODE REVIEW

File: code/game/gamemodes/revolution/revolution.dm [hunk 1]
```diff
@@ -151,9 +154,17 @@
 		mob << "The chameleon security HUD in your [where2] will help you keep track of who is loyalty-implanted, and unable to be recruited."
 
 	if (!where)
-		mob << "The Syndicate were unfortunately unable to get you a flash."
+		mob << "Unfortunately, due to budget cuts, the syndicate were unable to get you a conversion tool.  Maybe try prayer?."
+	else
+		mob << "The suspecious device in your [where] will help you to persuade the crew to join your cause."
```
Comment (oisin100) [inline code/game/gamemodes/revolution/revolution.dm]: *Suspicious

Comment (Super3222) [inline code/game/gamemodes/revolution/revolution.dm]: Every time Oisin, everrrrry timeeeee.

File: code/game/gamemodes/revolution/revolution.dm [hunk 2]
```diff
@@ -151,9 +154,17 @@
...
-		mob << "The Syndicate were unfortunately unable to get you a flash."
+		mob << "Unfortunately, due to budget cuts, the syndicate were unable to get you a conversion tool.  Maybe try prayer?."
+	else
+		mob << "The suspicious device in your [where] will help you to persuade the crew to join your cause."
+
+	if(!where3)
+		mob << "The Syndicate were unable to get you a hijack tool."
 	else
-		mob << "The flash in your [where] will help you to persuade the crew to join your cause."
+		mob << "The hijack tool that resembles an implanter in your [where3] is designed to create a conversion tool out of NT's flash devices. It can be startd by holding both items in your hand, and plugging the implanter into the flash."
```
Comment (oisin100) [inline code/game/gamemodes/revolution/revolution.dm]: *started

File: code/game/gamemodes/revolution/revolution.dm [hunk 3]
```diff
@@ -151,9 +154,17 @@
 		mob << "The chameleon security HUD in your [where2] will help you keep track of who is loyalty-implanted, and unable to be recruited."
 
 	if (!where)
-		mob << "The Syndicate were unfortunately unable to get you a flash."
+		mob << "Unfortunately, due to budget cuts, the syndicate were unable to get you a conversion tool.  Maybe try prayer?."
```
Comment (oisin100) [inline code/game/gamemodes/revolution/revolution.dm]: *Maybe try praying?

Comment (Super3222) [inline code/game/gamemodes/revolution/revolution.dm]: Eh, that was Mac's fault.

**Comment by MacHac** [comment]
Cannot into speelin.

**Pull request merged**